code/game/objects/items/devices/PDA/PDA_types.dm

```diff
@@ -161,7 +161,7 @@
 	name = "shaft miner PDA"
 	icon_state = "pda-miner"
 
-/obj/item/pda/syndicate
+/obj/item/pda/syndicate/nukeop
```

Comment:
you are making pda/syndicate an empty path that does nothing, why?

Reply:
it was a test to see if i could make a detatomix-less syndie pda but it didn't work and clearly, forgot to delete it